Hestia/Hestia/AppDelegate.cs

```diff
@@ -11,74 +11,48 @@
 
 namespace Hestia
 {
-    // The UIApplicationDelegate for the application. This class is responsible for launching the 
+    /// <summary>
+    /// The UIApplicationDelegate for the application. This class is responsible for launching the 
     // User Interface of the application, as well as listening (and optionally responding) to 
     // application events from iOS.
+    /// </summary>
     [Register("AppDelegate")]
     public class AppDelegate : UIApplicationDelegate
     {
-        // class-level declarations
-
-        public override UIWindow Window
-        {
-            get;
-            set;
-        }
-
+        public override UIWindow Window { get; set; }
         public static UIStoryboard mainStoryboard = UIStoryboard.FromName(strings.mainStoryBoard, null);
         public static UIStoryboard devices2Storyboard = UIStoryboard.FromName(strings.devices2StoryBoard, null);
+        public static UIStoryboard devices3Storyboard = UIStoryboard.FromName(strings.devices3StoryBoard, null);
 
         string defaultServername;
         string defaultIP;
         string defaultPort;
         string defaultAuth0AccessToken;
-        string defaultAuth0IdentityToken;
 
         public bool IsServerValid()
         {
+            if (defaultIP == null)
+            {
+                return false;
+            }
             try
             {
-                bool validIp = PingServer.Check(defaultIP, int.Parse(defaultPort));
+                bool validIp = PingServer.Check("https://" + defaultIP, int.Parse(defaultPort));

```

Comment:
`defaultIP` should already have `https://` or `http://` included, since a server can also use http.